_config.yml

```
@@ -1,15 +1,13 @@
-title: Your awesome title
-author: GitHub User
-email: your-email@domain.com
+title: This is Malind's Site
+author: M Kowalski  
+email: malind.nikulski@gmail.com
 description: > # this means to ignore newlines until "baseurl:"
```

Hi @mekowalski! I would try this. 🤔 I'm not 💯 sure if it's the problem but I have a feeling it might be. 

```suggestion
description: Welcome to this! I'm learning so much with Github.
```